api/serverpb/server.proto

```diff
@@ -194,6 +194,8 @@ message Settings {
   bool backup_management_enabled = 18;
   // True if Azure Discover is enabled.
   bool azurediscover_enabled = 17;
+  // True if the PMM instance is connected to Portal
+  bool connected_to_portal = 19;
```

Comment:
nitpick: maybe we should call this `connected_to_platform` as everywhere else it mentioned as platform

Reply:
@palash25 The platform is not software per se, but the Portal is. We connect PMM to the Portal. By the way, we launch under portal.percona.com.